src/foremast/awslambda/awslambda.py

```diff
@@ -61,6 +61,22 @@ def _check_lambda(self):
             pass
         return exists
 
+    def _check_lambda_alias(self):
+        """ Checks if lambda alias exists
+        Returns:
+            True if alias exists
+            False if alias does not exist
+        """
+        aliases = self.lambda_client.list_aliases(FunctionName=self.app_name)
+
+        for alias in aliases['Aliases']:
+            if alias['Name'] == self.env:
+                LOG.info('Found alias %s for function %s', self.env, self.app_name)
+                return True
```

Comment:
I am just curious why is this "more preferred" than having it the old way? For me at least old way is more readable, and easier to track. Also function is not really complex to warrant having only one exit point. I understand the point about multiple log statements but still think old way is more readable even though there are 2 log statements. I can change it, no problem, just trying to learn why it would be better :)